docs/user-guide/lns-discovery.md

```diff
@@ -0,0 +1,78 @@
+
+The discovery service relies on configuration from IoT Hub module twins to associate each LBS with a set of LNS to which it may connect to. For this the configuration needs to take into account possible network boundaries - by defining these networks you can control which LBS should connect to which LNS. For this you need to configure your LBS twin in IoT Hub as follows:
+
+```json
+{
+  ...,
+  "tags": {
+    "network": "<network-id>"
+  },
+  ...
+}
+```
+
+The network ID may only consist of alphanumeric characters. After you configured the LBS, you need to also configure your LNS with the same network. The LNS twin furthermore needs to contain the IP/DNS of the host, hence the twin of the LNS *module* should look like:
+
+!!! warning
+    We will only query the LNS modules for LNS within the same network. If you configure the network/host address on the IoT Edge device twin, it will not be detected.
+
+```json
+{
+  ...,
+  "tags": {
+    "network": "<network-id>"
+  },
+  ...,
+  "properties": {
+    "desired": {
+      "hostAddress": "wss://mylns:5001",
+      ...
+    }
+  }
+}
+```
+
+Keep in mind that the `hostAddress` value must include the scheme and the host name. In case you omit the port, the defaults of the protocol will be used.
+
+!!! note
+    The discovery endpoint distributes LBS to LNS using a round-robin distribution mechanism. It will always try the same LNS per LBS first. Hence, it can be used for an active/passive scenario, but not to distribute load more evenly across different LNS.
+
+!!! note
+    The configuration values of the network name of a station, respectively of the set of LNS in a given network, are cached by the discovery endpoint for 6 hours.
+    If you updated the configuration and want it to take effect sooner, make sure to restart the discovery service to ensure that the cache is refreshed.
+
+You can choose to deploy the discovery service either on-prem or in the cloud. For both deployment strategies, you can configure the following behavior:
+
+## Configuration
+
+You can configure the following behavior of the LNS discovery service.
+
+| Environment variable | Description | Deployment type |
+| ---                  | ---         | ---             |
+| `Logging__LogLevel__Default` | Configures the default log level. For more fine-grained configuration of the console log level, refer to [Logging in .NET Core and ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-6.0#set-log-level-by-command-line-environment-variables-and-other-configuration) | All |
+| `APPINSIGHTS_INSTRUMENTATIONKEY` | Sends telemetry data to Application Insights | All |
+| `Logging__ApplicationInsights__LogLevel__<Default|..>` | Configures the Application Insights log levels | All |
```

Comment:
[nit] I think this might not render correctly in a table and whaterver is after '|' would end up in a separate column. The same goes for line 63.

Reply:
Not sure I can reproduce, when rendering locally it looks correct to me:

![image](https://user-images.githubusercontent.com/22341213/155284598-27557800-b9ad-42d0-b55c-0f5bae9ae381.png)

Which renderer are you using?

Reply:
It was split into different columns here in the PR, when selecting the preview for the file. But since it's rendering as expected in the documentation page, then it's not a problem.